source/release-notes.html.md

```diff
@@ -5,6 +5,16 @@ weight: 200
 
 # Release notes
 
+### Unreleased Changes
+- Add Development, Test and Vendor Sandbox enviroment details to [api info](/reference/#api-info) page.
+- Add [/test-data/regenerate](/reference/#post-test-data-regenerate) endpoint.
+- Add [single application](/reference/#singleapplicationresponse) and [multiple applications](/reference/#multipleapplicationsresponse) response schemas.
+- Add `422` error response to endpoints using the `meta` field including:
```

Comment:
`422` applies to all `POST` requests, not just those using `meta` (although  this is all of them at the moment)

Reply:
Sounds good 👍